.changeset/wild-seas-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/vercel': major
```

Comment:
i think this is a patch. even though it feels like a big change there isn't a reason to believe it's going to break anyone's code, or is there?

Reply:
I agree, that it can be a patch, since we know it is working well for cloudflare there should be little risk to make it a patch release.